src/services/routes/index.js

```diff
@@ -20,6 +20,8 @@ router.get("/api/v1/routes", (req, res, next) => {
 });
 
 router.post("/api/v1/routes", (req, res, next) => {
+<<<<<<< HEAD
```

Comment:
These are leftover from a merge error. Please delete and review your merge to make sure you did it correctly.